media-libs/libsdl2/patches/libsdl2-2.32.10.patchset

```diff
@@ -324,5 +295,41 @@ index 38863ab..36cd2f1 100644
  
  static int _InitWindow(_THIS, SDL_Window *window) {
 -- 
-2.43.2
+2.51.0
+
+
+From 48cd7880ff983d863eacf7e43427a76afb5ca352 Mon Sep 17 00:00:00 2001
+From: Peppersawce <michaelpeppers89@yahoo.it>
+Date: Fri, 5 Dec 2025 04:15:59 +0100
+Subject: Make BUrl not 'ambiguous'
+
+
+diff --git a/src/misc/haiku/SDL_sysurl.cc b/src/misc/haiku/SDL_sysurl.cc
+index 5183234..341097c 100644
+--- a/src/misc/haiku/SDL_sysurl.cc
++++ b/src/misc/haiku/SDL_sysurl.cc
+@@ -24,7 +24,7 @@
+ 
+ int SDL_SYS_OpenURL(const char *url)
+ {
+-    BUrl burl(url);
++    BUrl burl(url, true);
+     const status_t rc = burl.OpenWithPreferredApplication(false);
+     return (rc == B_NO_ERROR) ? 0 : SDL_SetError("URL open failed (err=%d)", (int) rc);
+ }
+diff --git a/src/video/haiku/SDL_bvideo.cc b/src/video/haiku/SDL_bvideo.cc
+index d541289..8b7e094 100644
+--- a/src/video/haiku/SDL_bvideo.cc
++++ b/src/video/haiku/SDL_bvideo.cc
+@@ -305,7 +305,7 @@ void HAIKU_VideoQuit(_THIS)
+ extern "C" { int HAIKU_OpenURL(const char *url); }
+ int HAIKU_OpenURL(const char *url)
+ {
+-    BUrl burl(url);
++    BUrl burl(url, true);
+     const status_t rc = burl.OpenWithPreferredApplication(false);
+     return (rc == B_NO_ERROR) ? 0 : SDL_SetError("URL open failed (err=%d)", (int)rc);
+ }
+-- 
+2.51.0
```

Comment:
This patch is for nightly only. The builders (except maybe riscv64) still run beta 5, so this will not work there.